modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -241,6 +248,13 @@ struct IEUnit {
         if (params.num_out == 1u && params.output_names.empty()) {
             params.output_names = { outputs.begin()->first };
         }
+        if (!params.reshape_table.empty()) {
+            input_reshape_table.clear();
```

Comment:
It isn't necessary, is it ? You anyway do assignment  to `input_reshape_table`

Reply:
Yes.